provisioning/service/samples/Getting Started/IndividualEnrollmentTPMSample/Properties/launchSettings.json.template

```diff
@@ -0,0 +1,10 @@
+{
+    "profiles": {
```

Comment:
same remove.

We don't need launchSettings templates. We move instead to use parameters (CommandLineParser library). See other samples for examples.